superset/assets/javascripts/explorev2/components/Control.jsx

```diff
@@ -48,6 +48,7 @@ export default class Control extends React.PureComponent {
     super(props);
     this.validate = this.validate.bind(this);
     this.onChange = this.onChange.bind(this);
+    this.onChange(props.value, []);
```

Comment:
i think this might be more clear if we make a new method to render the validation errors that can be called in the constructor and in onChange, rather than calling onChange in the constructor.